src/DoctrineSnapshotAdapter.php

```diff
@@ -109,6 +109,13 @@ public function add(Snapshot $snapshot)
                 'blob',
             ]
         );
+
+        $this->connection->executeQuery(
```

Comment:
We should use the query builder also for delete queries as table and column quoting can be different for the various database vendors